### ISSUE TITLE
Tech-radar not showing all items

### ISSUE THREAD

**Issue by jvilimek** [backstage/backstage#5693]
We are using [tech-radar](https://github.com/backstage/backstage/tree/master/plugins/tech-radar) to provide hints for our developers what to use and where to find more info (links for wiki). The list of technologies/tools is quite comprehensive. We have about 45 tools. Some of them are not visible in the list as obvious from the screenshot below:

![image](https://user-images.githubusercontent.com/16001792/118486626-1be5a680-b71a-11eb-98d0-6f1a019d69d5.png)

## Expected Behavior

All items visible or some scrollbar visible or some paging available

## Current Behavior

If lot of items some of them are not visible. May be connected to the screen resolution or lenght of the titles as well (word wrap)

## Possible Solution

Add scrollbar or pagination

## Steps to Reproduce

Put following list amond one quadratant in tech-radar:
```
Azure   Devops
Azure   AppInsights
Azure   KeyVault
Azure Storage
SQL Azure
Azure Cosmos   DB
Azure Redis
Azure Active   Directory
Azure ML   Studio
Active   Directory
Pentaho
Azure Data   Factory
Crystal   Reports
Zabbix
Azure API   Management
Backstage
Azure   EventHub &amp; Service Bus
Oracle   database
Oracle   Exadata
Web   Application Firewall - F5
Ansible
LucidWorks   Fusion
MailChimp
MindBox
SalesManago
SendGrid
PowerBI
Azure   Analysis Services
Cloudinary
Google Dialog   Flow
Google AutoML   Tables
Cornerstone
Cognos BI
TM1
Google Data   Studio
Google   Analytics
Google Big   Query
Google Vision   AI
Google Cloud   Natural Language
Azure   Cognitive Services
Visual Studio   Code
Visual Studio
Lerna
Storybook
Lighthouse
```

## Your Environment

- Browser Information: Chrome v90
- tech-radar v0.3.11

**Comment by freben** [comment]
Yeah, reproduced. Not sure how best to render that in an effective way - it's constrained to stay within the quadrant space available so it just plain runs out of room. And as you say some form of paging or scrolling will probably be necessary. It uses the CSS `columns` prop and some breaking rules; will probably require some fiddling to look good both for few and for many items.

**Comment by jvilimek** [comment]
Please any update?

**Comment by jhaals** [comment]
The issue is not picked up by anyone at this point. It would be fantastic if you would be willing to help the project by contributing a fix to this issue.

**Comment by jvilimek** [comment]
Unfortunatelly my typescript/css skills are not great... I am more C# and backend oriented :-(

**Comment by o-farooq** [comment]
@jvilimek  Not a proper solution , just a workaround until fixed properly is to change the TechRadarPage size in App.tsx . 1800x1150 worked for me 

![image](https://user-images.githubusercontent.com/17722640/125181693-8539ef80-e25b-11eb-8561-1f952f890921.png)
